tests/unit/components/ont/runs/ONTFlowcell.spec.js

```diff
@@ -98,6 +98,11 @@ describe('ONTFlowcell', () => {
         ontFlowcell.setPoolTubeBarcode({ barcode: 'TRAC-1-B', position: 1 })
         expect(ontFlowcell.poolTubeBarcode).toEqual('TRAC-1-B')
       })
+
+      it('#setNewFlowCell', () => {
```

Comment:
makes sense to add the test but currently failing CI

Reply:
Sorry, I didn't get it. Why are we adding this test?

Reply:
Oh, ok. Thank you for explaining the context. 👍

Reply:
Hmm, does it pass locally?

Reply:
I had a quick look. I think the position is a prop and component is mounted with a position 1. The setNewFlowCell is a store method which creates a new flowcell here at position 2., But, I suppose it won't change the position value of an already existing component.